.github/workflows/test-molecule.yml

```diff
@@ -309,7 +311,7 @@ jobs:
           # {role: "manage-service", test: "mevboost-besu-nimbus"},
           # {role: "manage-service", test: "mevboost-besu-prysm"},
           # {role: "manage-service", test: "mevboost-besu-teku"},
-          # {role: "manage-service", test: "mevboost-geth-lighthouse"},
+          # {role: "manage-service", test: "mevboost-geth-lodestar"},
```

Comment:
I'd suggest to add `mevboost-geth-lodestar` while still leaving `mevboost-geth-lighthouse` in place (not replace it).

Reply:
done!